hw2/05-form.js

```diff
@@ -1,15 +1,43 @@
 // Add your code here
-const userName = document.getElementById('InputName').value
 const submit = document.getElementById('Submit');
 const reset = document.getElementById('Reset');
 
+submit.addEventListener('click', ()=>{
 
-console.log(userName)
+    const userName = document.getElementById('InputName').value
+    const email = document.getElementById('InputEmail').value
+    const registration = document.getElementById('Registration').value
+    const progCheck = document.getElementById('ProgrammingLang').checked
+    const opCheck = document.getElementById('OperatingSystems').checked
+    const fullCheck = document.getElementById('FullStack').checked
+    const anything = document.getElementById('Anything').value
+    console.log(userName)
+    console.log(email)
+    console.log(registration)
 
-submit.addEventListener('click', ()=>{
```

Comment:
I would suggest exploring the “submit” event.  Although in this case, click event is working out, submit is usually considered as a good practice!